data/src/test/java/io/gnosis/data/models/TransactionDtoTest.kt

```diff
@@ -0,0 +1,76 @@
+package io.gnosis.data.models
+
+import com.squareup.moshi.JsonDataException
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import io.gnosis.data.adapters.OperationEnumAdapter
+import io.gnosis.data.db.BigDecimalNumberAdapter
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import pm.gnosis.common.adapters.moshi.MoshiBuilderFactory
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.util.stream.Collectors
+
+class TransactionDtoTest {
+
+    private var moshi = MoshiBuilderFactory.makeMoshiBuilder()
+        .add(BigDecimalNumberAdapter())
+        .add(OperationEnumAdapter())
+        .add(transactionDtoJsonAdapterFactory)
+        .add(KotlinJsonAdapterFactory())
+        .build()
```

Comment:
This will definitely fall out of sync with the one we use in the `ApplicationModule` I would say we push down the provides method for dagger into `:data` so we use the same object as we do in runtime.